resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh

```diff
@@ -67,6 +67,14 @@ elif ! [ -z ${SPARK_HOME+x} ]; then
   SPARK_CLASSPATH="$SPARK_HOME/conf:$SPARK_CLASSPATH";
 fi
 
+# when user want to formulate the IP they need, they can use this conf to post host
+if ! [ -z "${SPARK_EXECUTOR_POD_IP_OVERWRITE}" ]; then
```

Comment:
What is this `SPARK_EXECUTOR_POD_IP_OVERWRITE`?

Reply:
it is a executor pod env, maybe we should describe it alone like ENV_DRIVER_URL？